src/marketing/price-rule-multiple-sku.md

```diff
@@ -2,53 +2,56 @@
 title: Price Rule with Multiple SKUs
 ---
 
-A single price rule can be applied to multiple SKUs, which makes it possible to create a variety of promotions based on a product, brand, or category. When creating this rule, you want to set conditions to match any products of selected SKUs. When building the rule, you can easily browse and select SKUs without needing to know all of your product SKUs from memory.
+A single catalog price rule can be applied to multiple SKUs, which makes it possible to create a variety of promotions based on a product, brand, or category. When creating this rule, you want to set conditions that match the selected SKUs. When building the rule, you can easily browse and select SKUs from the grid.
 
 ## Step 1. Verify Storefront Properties of Attribute
 
-Before you begin, make sure that the Storefront Properties of the SKU attribute are set to `Use in Promo Rules`.
+Before you begin, make sure that the Storefront Properties of the `sku` attribute are set to `Use in Promo Rules`.
```

Comment:
it might be helpful to link to add a link to the storefront properties in the attribute topic.   i have not been able to get an internal link for a .md file to link to a header, but you might use the html link

storefront properties - (https://docs.magento.com/m2/ee/user_guide/catalog/product-attributes-add.html#step-5-describe-the-storefront-properties)

Reply:
Fixed.